### PR TITLE
FIX #119 - better messages during reporting in CLI

### LINKED ISSUE #119: Reporting tweaks [Alpha Release]
Testing the new alpha release as part of 2.0, I found a couple of nitpicks/tweaks:

1. Regarding the following output:
chromatic info Building your Storybook

It doesn't make sense to say "Uploading and verifying build..." after we've said "Uploaded your build, verifying"

2. We need to update the docs url to `https://docs.chromatic.com`
chromatic info Pass --exit-zero-on-changes if you want this command to exit successfully in this case

3. We should finally fix the fact that it says "Build 1 has 3 changes" for the first build:

image

4. At the end it says:

```
NOTE: I wrote your project token to the script via the \`--projectToken\` flag.
```

Isn't it `--project-token`? In any case it just wrote `"chromatic": "chromatic"`.

### PR DESCRIPTION
**Pull request by ndelangen**
fixes #119

### CODE REVIEW

File: bin/tester/index.js [hunk 1]
```diff
@@ -603,7 +600,7 @@ export async function runTest({
         dedent`
           Added script 'chromatic'. You can now run it here or in CI with 'npm run chromatic' (or 'yarn chromatic')
 
-          NOTE: I wrote your project token to the script via the \`--projectToken\` flag. 
+          NOTE: I wrote your project token to the script via the \`--project-token\` flag. 
           
           The project token cannot be used to read story data, it can only be used to create new builds.
           If you would still prefer not to check it into source control, you can remove it from 'package.json' and set it via the \`CHROMATIC_PROJECT_TOKEN\` environment variable instead in your CI environment.
```
Comment (domyen) [inline bin/tester/index.js:606]:
```suggestion
          If you're running Chromatic via continuous integration, we recommend setting \`CHROMATIC_PROJECT_TOKEN\` environment variable in your CI environment. You can then remove the --project-token from your 'package.json'.
```

File: bin/tester/index.js [hunk 2]
```diff
@@ -603,7 +600,7 @@ export async function runTest({
         dedent`
           Added script 'chromatic'. You can now run it here or in CI with 'npm run chromatic' (or 'yarn chromatic')
 
-          NOTE: I wrote your project token to the script via the \`--projectToken\` flag. 
+          NOTE: I wrote your project token to the script via the \`--project-token\` flag.
```
Comment (domyen) [inline bin/tester/index.js:603]:
```suggestion
          NOTE: Your project token was added to the script via the \`--project-token\` flag. 
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -211,7 +211,7 @@
 
 - Better support for rebasing branches - we now always treat the last build on this branch as a baseline, even if strictly it is not a git ancestor of the current commit. This helps deal with the situation where you rebase a branch off master, and still want to use the previously approved snapshots.
 
-- Improved support for CI systems, especially _Netlify_ and _Travis PR builds_. Travis PR builds are a special case, read more about how to handle them in Chromatic here: https://docs.chromaticqa.com/setup_ci#travis
+- Improved support for CI systems, especially _Netlify_ and _Travis PR builds_. Travis PR builds are a special case, read more about how to handle them in Chromatic here: https://docs.chromatic.com/setup_ci#travis
```
Comment (zol) [inline CHANGELOG.md:214]: @ndelangen at least the `/setup_ci` path has changed (it's now just `/ci`). Please check the urls you link to against the latest docs here: https://github.com/chromaui/chromatic-docs/tree/edits . The names of the md files just map to /name

Comment (tmeasday) [reply CHANGELOG.md:214]: @zol are you going to make the old URLs redirect?

**Comment by zol** [comment]
> @zol are you going to make the old URLs redirect?

I wasn't planning to because I did't think the cost vs benefit is worth it but I can revisit that decision if you disagree!

**Comment by tmeasday** [comment]
Well in particular older versions of the CLI will have the old links, so we probably need to?

**Comment by zol** [comment]
Ahh ok. I've created a TODO on the docs ticket and will create the redirects

File: CHANGELOG.md [hunk 4]
```diff
@@ -65,7 +65,7 @@
 
 - Add support for docs-mode (Storybook 5.2 feature).
 
-- Add support for a new parameter: `pauseAnimationsAtEnd`. Read about it here: https://docs.chromaticqa.com/animations
+- Add support for a new parameter: `pauseAnimationsAtEnd`. Read about it here: https://docs.chromatic.com/animations
```
Comment (tmeasday) [inline CHANGELOG.md:68]:
```suggestion
- Add support for a new parameter: `pauseAnimationsAtEnd`. Read about it here: https://www.chromatic.com/docs/animations
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -211,7 +211,7 @@
 
 - Better support for rebasing branches - we now always treat the last build on this branch as a baseline, even if strictly it is not a git ancestor of the current commit. This helps deal with the situation where you rebase a branch off master, and still want to use the previously approved snapshots.
 
-- Improved support for CI systems, especially _Netlify_ and _Travis PR builds_. Travis PR builds are a special case, read more about how to handle them in Chromatic here: https://docs.chromaticqa.com/setup_ci#travis
+- Improved support for CI systems, especially _Netlify_ and _Travis PR builds_. Travis PR builds are a special case, read more about how to handle them in Chromatic here: https://docs.chromatic.com/ci#travis
```
Comment (tmeasday) [inline CHANGELOG.md:214]:
```suggestion
- Improved support for CI systems, especially _Netlify_ and _Travis PR builds_. Travis PR builds are a special case, read more about how to handle them in Chromatic here: https://www.chromatic.com/docs/ci#travis
```

File: bin/git/getCommitAndBranch.js [hunk 6]
```diff
@@ -41,7 +41,7 @@ export async function getCommitAndBranch({ patchBaseRef, inputFromCI } = {}) {
 
         It is recommended to run Chromatic on the push builds from Travis where possible.
         We advise turning on push builds and disabling Chromatic for internal PR builds.
-        Read more: https://docs.chromaticqa.com/setup_ci#travis
+        Read more: https://docs.chromatic.com/ci#travis
```
Comment (tmeasday) [inline bin/git/getCommitAndBranch.js:44]:
```suggestion
        Read more: https://www.chromatic.com/docs/ci#travis
```

File: bin/git/getCommitAndBranch.js [hunk 7]
```diff
@@ -59,7 +59,7 @@ export async function getCommitAndBranch({ patchBaseRef, inputFromCI } = {}) {
       \`TRAVIS_EVENT_TYPE\` environment variable set to '${TRAVIS_EVENT_TYPE}', 
       but \`TRAVIS_PULL_REQUEST_SHA\` and \`TRAVIS_PULL_REQUEST_BRANCH\` are not both set.
       
-      Read more here: https://docs.chromaticqa.com/setup_ci#travis
+      Read more here: https://docs.chromatic.com/ci#travis
```
Comment (tmeasday) [inline bin/git/getCommitAndBranch.js:62]:
```suggestion
      Read more here: https://www.chromatic.com/docs/ci#travis
```

File: bin/git/getCommitAndBranch.js [hunk 8]
```diff
@@ -72,7 +72,7 @@ export async function getCommitAndBranch({ patchBaseRef, inputFromCI } = {}) {
         \`GITHUB_WORKFLOW\` environment variable set to '${GITHUB_WORKFLOW}', 
         but \`GITHUB_SHA\` and \`GITHUB_REF\` are not both set.
 
-        Read more here: https://docs.chromaticqa.com/setup_ci#github
+        Read more here: https://docs.chromatic.com/ci#github
```
Comment (tmeasday) [inline bin/git/getCommitAndBranch.js:75]:
```suggestion
        Read more here: https://www.chromatic.com/docs/ci#github
```

File: bin/tester/index.js [hunk 9]
```diff
@@ -545,7 +544,7 @@ export async function runTest({
           log.info(dedent`
             Pass --exit-zero-on-changes if you want this command to exit successfully in this case.
             Alternatively, pass --auto-accept-changes if you want changed builds to pass on this branch.
-            Read more: https://docs.chromaticqa.com/test
+            Read more: https://docs.chromatic.com/test
```
Comment (tmeasday) [inline bin/tester/index.js:547]:
```suggestion
            Read more: https://www.chromatic.com/docs/test
```

**Pull request merged**